contentrepo/settings/production.py

```diff
@@ -6,3 +6,20 @@
     from .local import *
 except ImportError:
     pass
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.environ.get('DATABASE_NAME','name'),
+        'USER': os.environ.get('DATABASE_LOGIN','user'),
+        'PASSWORD': os.environ.get('DATABASE_PASSWORD', 'password'),
+        'HOST': os.environ.get('DATABASE_HOST', ''),
+        'PORT': '5432',  # Set to empty string for default.
+        'CONN_MAX_AGE': 600,
+    }
```

Comment:
I think we usually do this by installing dj_database_url and then doing it like https://github.com/praekeltfoundation/casepro-docker/blob/master/settings.py#L27-L31
but this is fine for now